environment.yml

```diff
@@ -8,3 +8,4 @@ dependencies:
   - samtools==1.8
```

Comment:
Is samtools used in this pipeline? Or does Picard do al the sorting?

Reply:
For star it is required. After this changes samtools is not required anymore for bwa mem

Reply:
Will remove the import from the pipeline